### DISCUSSION TITLE
Welcome to 777-Interview-Notes Discussions!

### DISCUSSION THREAD

**Discussion by hishark** [hishark/777-Interview-Notes#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hishark** [comment]
诶嘿这个新功能不错诶，比开 issue 去讨论问题好多啦！

**Comment by hishark** [comment]
最近忙着准备一些东西，复习笔记暂时杂乱的在本地存储，等忙完这个月之后整理好就放到站子里去 > <

**Reply by sakurafisch** [reply]
可以考虑写个脚本每次运行就自动执行git add/commit/push，这样子比较方便

**Reply by hishark** [reply]
<img src="https://user-images.githubusercontent.com/29684201/104126458-a185ce00-5397-11eb-8f45-cb75851985e1.png" width="300px">

新版本的gitbook是直接可以编辑保存的已经非常方便啦，我是因为前阵子比较忙笔记单纯给自己看所以内容上比较杂乱，所以是想整理好内容之后再更新到站子里哈哈，谢谢建议～😉

**Reply by sakurafisch** [reply]
soga  我在此学习到了很多知识，非常期待下次的更新~

**Reply by hishark** [reply]
哈哈好的呀，我也在学习！跟着 CYC 大佬的笔记边学边查漏补缺 一起加油～

**Reply by sakurafisch** [reply]
CYC 大佬 is who?!

**Reply by hishark** [reply]
就是我的参考链接里的呀，这个这个！[http://www.cyc2018.xyz/](http://www.cyc2018.xyz/)

**Reply by sakurafisch** [reply]
哈哈哈哈好的好的，干巴爹！

**Comment by sakurafisch** [comment]
作为一个有 watch 仓库的人还以为你给我发邮件了（x

**Reply by hishark** [reply]
哈哈哈哈哈哈这个有点意思

**Comment by sakurafisch** [comment]
你在Java常见问题中提到，synchronized 可修饰变量、方法和类。但 synchronized 似乎不可修饰变量。

**Reply by hishark** [reply]
嗯嗯对的！那句话有问题我已经删掉啦
谢谢抓虫！！=v=

**Comment by wencaiHua** [comment]
怎样才能坚持刷题呢？

**Reply by hishark** [reply]
我觉得可以从每日一题开始哦，力扣的每日一题活动我觉得超好哈哈，题解也都特别详细。
